### PR TITLE
Added support for more EPSG codes, added test cases 

### PR DESCRIPTION
**Pull request by SaucySells**

### CODE REVIEW

File: pyfite/coordinates.py [hunk 1]
```diff
@@ -463,14 +464,26 @@ def __repr__(self):
...
+        return isinstance(other, ProjCrs) and self._proj == other._proj and self._offset == other._offset  # pylint: disable=protected-access
 
     @staticmethod
     def from_str(srep: str) -> 'ProjCrs':
         """See ``CoordinateReferenceSystem.from_str``
         """
         return ProjCrs(srep)
 
+    @staticmethod
+    def from_epsg(code: Union[str,int], offset: Tuple[float, float, float]) -> 'ProjCrs':
```
Comment (rdhite) [inline pyfite/coordinates.py]: offset should be optional and default to (0, 0, 0) as it does in the constructor

File: pyfite/_tests/test_coordinates.py [hunk 2]
```diff
@@ -413,3 +413,16 @@ def test_utm_from_point():
...
+
+
+def test_projcrs_from_epsg():
+    """Tests that a proj string is properly built from EPSG codes.
+    """
+    epsg4326_crs = pfc.ProjCrs.from_epsg(4326, (60.25, -80.12, 0))
+    epsg4978_crs = pfc.ProjCrs.from_epsg(4978, (34.09, -118.13, 0))
+    epsg32618_crs = pfc.ProjCrs.from_epsg(32618, (28.54, -81.38, 0))
+
+    assert str(epsg4326_crs) == '+proj=longlat +datum=WGS84 +no_defs +type=crs 60.25 -80.12 0'
```
Comment (rdhite) [inline pyfite/_tests/test_coordinates.py]: use the new assert_proj_str_equivalent method

File: pyfite/coordinates.py (hunk 1 above)
Comment (SaucySells) [reply pyfite/coordinates.py]: Fixed

File: pyfite/_tests/test_coordinates.py (hunk 2 above)
Comment (SaucySells) [reply pyfite/_tests/test_coordinates.py]: Fixed, also started using that method in test_context_capture

File: pyfite/_tests/testdata/metadata_epsg4326_offset.xml [hunk 3]
```diff
@@ -0,0 +1,2 @@
+<SRS>EPSG:4326</SRS>
+<SRSOrigin>1423,-1205,0</SRSOrigin>
```
Comment (rdhite) [inline pyfite/_tests/testdata/metadata_epsg4326_offset.xml]: I suppose this is fine for what the unit tests are doing at the moment, but these values are way outside the range of what you find in EPSG:4326. Bring these somewhere in the range of [-180,180), [-90, 90]. Altitude can be any number.

File: pyfite/coordinates.py [hunk 4]
```diff
@@ -463,14 +464,26 @@ def __repr__(self):
...
         """
         return ProjCrs(srep)
 
+    @staticmethod
+    def from_epsg(code: Union[str,int], offset: Optional[Tuple[float, float, float]] = (0.0, 0.0, 0.0)) -> 'ProjCrs':
+        """Creates ProjCrs from an EPSG code.
+
+        Args:
+            code: The EPSG code to use
+
```
Comment (rdhite) [inline pyfite/coordinates.py]: Pylint reports trailing whitespace here. Make sure to run `pylint pyfite` when pushing up. There should only be two TODOs in obj.py with my name on them.

File: pyfite/coordinates.py [hunk 5]
```diff
@@ -445,15 +445,16 @@ class ProjCrs(CoordinateReferenceSystem):
...
-    def __init__(self, proj_str: str, offset=(0.0, 0.0, 0.0)):
+    def __init__(self, proj_str: str, offset: Optional[Tuple[float, float, float]] = (0.0, 0.0, 0.0)):
         self._proj = proj_str
         self._offset = offset
 
     def __str__(self):
         """See ``CoordinateReferenceSystem.__str__``.
         """
-        return self._proj
+        return self._proj + self._get_offset_str()
```
Comment (rdhite) [inline pyfite/coordinates.py:459]: Per the documentation in CoordinateReferenceSystem, this method must produce a string that is ingestible by the `from_str` method to produce an equivalent CoordinateReferenceSystem. You'll have to update the ProjCrs.from_string to account for an optional offset at the end. You'll have to ensure that if there are three values at the end of the string, none of them are linked to a proj string argument before accepting them as an offset. Then we need tests to check that functionality.

File: pyfite/_tests/testdata/metadata_epsg4326_offset.xml (hunk 3 above)
Comment (SaucySells) [reply pyfite/_tests/testdata/metadata_epsg4326_offset.xml]: Changed to more reasonable values

File: pyfite/coordinates.py (hunk 4 above)
Comment (SaucySells) [reply pyfite/coordinates.py]: Fixed

File: pyfite/coordinates.py (hunk 5 above)
Comment (SaucySells) [reply pyfite/coordinates.py:459]: Updated from_str method to take offsets into account, added tests for it in test_coordinates.py

**Review by Seongioi — APPROVED** [review]
LGTM 🚢

**Review by rdhite — APPROVED** [review]

**Pull request merged**